src/main/java/com/groupsix/mapFood/pojo/OrderItem.java

```diff
@@ -18,6 +20,7 @@ public void setId(Integer id) {
 		this.id = id;
 	}
 	
+	@JsonAlias("product_id")

```

Comment:
Remover jsonAlias
padrão de json é camelCase
"Property names must be camel-cased, ascii strings."
https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format